src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard.test.ts

```diff
@@ -70,16 +77,21 @@ const testStep2HaveBillingChecked = () => {
   verifyUnchecked(getStep2BillingAccountNoAccessButton())
 }
 
-const testStep3InitialState = () => {
+// the happy path
+const testAddTerraUserRenderedForStep3 = () => {
+  expect(getStep3VerifyUserAdded()).toBeNull()
+  expect(getStep3BillingAccountNoAccessButton()).not.toBeNull()
+  expect(getStep3AddedTerraBillingButton()).not.toBeNull()
+}
+
+const testAddTerraUserStep3UninitializedState = () => {
   verifyDisabled(getStep3BillingAccountNoAccessButton())
-  verifyDisabled(getStep3AddedTerraBillingButton())
   verifyUnchecked(getStep3BillingAccountNoAccessButton())
+  verifyDisabled(getStep3AddedTerraBillingButton())
   verifyUnchecked(getStep3AddedTerraBillingButton())
-  expect(getStep3AddTerraAsUserText()).not.toBeNull()
```

Comment:
Why remove these asserts?

Reply:
TLDR; Going through my git history, it looks like it was mostly a mistake. 

Long form:
The intended result was to split up testStep3InitialState into two separate functions: one that was testing which version of step 3 was rendered (testAddTerraUserRenderedForStep3), and the other that was testing the state of the inputs (testAddTerraUserStep3UninitializedState). 
I think by the time I got to that point, I had been staring at the code for too long, b/c I missed a step in my process.
Initially I had 4 functions: the two here, one that the other version of step 3 was rendered, and one checking the inputs on the other version. But it didn't make sense to use the other two functions, since the checks were already rolled into different places.

- `expect(getStep3VerifyUserAdded()).toBeNull()` was moved to `testAddTerraUserRenderedForStep3` (line 82, now)
- `expect(getStep3ContactBillingAdministrator()).toBeNull()` was one of the things I flipped the logic of, and put into the function testing that the other version of the component was rendered
- `expect(getStep3AddTerraAsUserText()).not.toBeNull()` .... just got missed, I think, and I didn't notice because I was still getting the signal that the component overall was being rendered 

I'll put the missing two back in the function that's testing that the component overall is rendered.


TY for catching that, and please excuse my stream of consciousness above.